### PR TITLE
Process data into expression and mutation matrices

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

File: execute.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (cgreene) [inline execute.sh:1]: reviewed - running conda environment

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+name: cognoma-cancer-data
```
Comment (cgreene) [inline environment.yml:1]: dependencies - looks reasonable

**Comment by cgreene** [comment]
The 2.TCGA-process.ipynb notebook says "Notice that mutation type codes are not standardized. For example, 5_prime_UTR_variant, 5'UTR, UTR_5_PRIME all refer to the same type of mutation." but the corresponding table doesn't have these elements. Did this get cleaned up before the table is generated somehow?

**Comment by clairemcleod** [comment]
It seems like there is a bug with the exported .yml file and environment creation. `execute.sh` does not actually create an environment. It seems related to [this conda issue](https://github.com/conda/conda-env/issues/247).

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+dependencies:
```
Comment (clairemcleod) [inline environment.yml:2]: Some of these package requirements are specific to conda on linux. When `execute.sh` runs on osx, yields error due to crossplatform issue:
Error: Packages missing in current osx-64 channels: 
- fontconfig 2.11.1 6
- libsodium 1.0.10 0
- zeromq 4.1.4 0

(edited for clarity)

**Comment by dhimmel** [comment]
> Did this get cleaned up before the table is generated somehow?

@cgreene see https://github.com/cognoma/cancer-data/issues/2#issuecomment-233088248 -- the input data changed, resolving the issue. Will update the notebook.

File: environment.yml (hunk 3 above)
Comment (dhimmel) [inline environment.yml:2]: @clairemcleod interesting error. It looks like `zeromq 4.1.4` is available for Linux, which I used to make the PR, but not for OS X in the [anaconda channel](https://anaconda.org/anaconda/zeromq/files?page=1). 4.1.3 is available for OS X.

It looks like others are having [similar issues](http://stackoverflow.com/q/38400676/4651668) recently. Options are to downgrade the versions for OS X compatibility, or to [notify anaconda](https://github.com/ContinuumIO/anaconda-issues/issues) and see if they can fix this.

Comment (clairemcleod) [inline environment.yml:2]: @dhimmel It seems like it is the same issue with anaconda channel availability for the other two as well:
[`libsodium`](https://anaconda.org/anaconda/libsodium/files) - libsodium 1.0.3 is the most recent version available on linux/windows/osx
[`fontconfig`](https://anaconda.org/anaconda/fontconfig/files) - only available for linux? not sure what to do about this

Comment (dhimmel) [inline environment.yml:2]: I submitted an issue to anaconda: https://github.com/ContinuumIO/anaconda-issues/issues/925. Let's see how that goes.

Comment (dhimmel) [inline environment.yml:2]: @clairemcleod, I attempted to fix the issue (see https://github.com/cognoma/cancer-data/pull/7/commits/8fb66e94fec1b1df48121f30b45c3a492082c0a3). Do you want to see whether the environment works for you on OS X now?

Comment (clairemcleod) [inline environment.yml:2]: @dhimmel This seems to work now - thanks!

**Comment by cgreene** [comment]
My comments have been addressed 👍

**Comment by dhimmel** [comment]
Cool @clairemcleod, would you like to merge if you think it's ready?

You'll want to use a "merge commit" rather than a "squash and merge" since there are multiple distinct commits. This is the default, but just wanted to clarify.

**Pull request merged**